test/e2e/persistent_volumes-vsphere.go

```diff
@@ -137,7 +141,6 @@ var _ = framework.KubeDescribe("PersistentVolumes:vsphere", func() {
 	It("should test that deleting a PVC before the pod does not cause pod deletion to fail on PD detach", func() {
```

Comment:
`PD` refers to a GCE persistent disk.  If it's not the same for vsphere, it will need to change.